test/ci/build_install-deps.xml

```diff
@@ -17,12 +17,22 @@
 	<target name="saxon">
 		<echo message="Install Saxon ${env.SAXON_VERSION}" />
 
+		<tempfile deleteonexit="true" destdir="${java.io.tmpdir}" prefix="saxon_deleteonexit_"
+			property="saxon.zip" suffix=".zip" />
+		<loadresource property="saxon.version.with.hyphen.replacing.dot">
+			<propertyresource name="env.SAXON_VERSION" />
+			<filterchain>
+				<tokenfilter>
+					<replaceregex flags="g" pattern="\." replace="-" />
+				</tokenfilter>
+			</filterchain>
+		</loadresource>
+		<download dest="${saxon.zip}"
+			src="${env.SAXON_URL}/SaxonHE${saxon.version.with.hyphen.replacing.dot}J.zip" />
+
 		<dirname file="${env.SAXON_JAR}" property="saxon.jar.dir" />
 		<mkdir dir="${saxon.jar.dir}" />
-
-		<download dest="${env.SAXON_JAR}"
-			src="https://repo1.maven.org/maven2/net/sf/saxon/Saxon-HE/${env.SAXON_VERSION}/Saxon-HE-${env.SAXON_VERSION}.jar"
-		 />
+		<unzip dest="${saxon.jar.dir}" src="${saxon.zip}" />
```

Comment:
Saxon 11 needs the XML Resolver (new kind: `xmlresolver*.jar`, not `resolver.jar`) jar file on the classpath, in addition to the main Saxon jar file. The download package from GitHub includes both jar files -- in the ZIP-file and in the classpath part of the manifest. If we get jar files from the Maven repository, we'd have to download multiple jar files individually and include the xmlresolver jar file on the classpath. (Note: the Saxon-HE-11.5.jar file from the Maven repo is not identical to the saxon-he-11.5.jar file from GitHub's ZIP-file, as they differ in the manifest. This difference is mentioned in https://saxonica.plan.io/boards/3/topics/8478?r=8868#message-8868 .) I believe that adapting the Maven approach would require more changes in XSpec repo code compared to getting the ZIP-file from GitHub and unzipping it.

Reply:
> the Saxon-HE-11.5.jar file from the Maven repo is not identical to the saxon-he-11.5.jar file from GitHub's ZIP-file, as they differ in the manifest.

This is a kind of critical info. Could you add it as a comment within this file?

Reply:
Good idea. Addressed in 0c0c96c2e1abb88e1754699af024bebd73cf0ff5.